web/Weblate-server.qmd

```diff
@@ -238,21 +238,27 @@ Weblate will provide a suggestion on how to fix, but in short:
     git svn rebase
     ```
 
-6. Optionally, if not yet done, install [helpers](https://github.com/JulienPalard/po3way) to make it easier to deal with the git conflicts (inside Docker):
+6. Optionally, if not yet done, install [helpers](https://github.com/JulienPalard/po3way) to make it easier to deal with the git conflicts (inside Docker). Start Docker as root to install `po3way` system-wide:
+
 
    ```sh
-   pip install po3way
+   $ sudo docker exec -u root -ti weblate-docker_weblate_1 bash
    ```
 
-   Then add this folder to `$PATH` or symlink it, e.g. starting Docker as root:
+   Then in the Docker container:
+
+   ```sh
+   # pip install --prefix /usr/local po3way
```

Comment:
Why `#`? I would omit this; below I see `$` used in a similar code block, so we should at least be consistent.

Reply:
`#` suggests that the prompt is for `root` and not a standard user